### PR TITLE
editorial cleanup

### PR DESCRIPTION
**Pull request by cem-**
I believe these are all non-controversial, editorial changes.

### CODE REVIEW

File: draft-ietf-tls-tls13.md [hunk 1]
```diff
@@ -4119,7 +4122,7 @@ CertificateVerify and certificates), and ecdsa_secp256r1_sha256. A
 TLS-compliant application MUST support key exchange with secp256r1
 (NIST P-256) and SHOULD support key exchange with X25519 {{RFC7748}}.
 
-##  MTI Extensions
+##  Mandatory-to-Implement Extensions
```
Comment (cem-) [inline draft-ietf-tls-tls13.md]: Changing this header breaks the ref on line 1907 (and is why the travis-ci broke). [So that will need fixing.]

Comment (martinthomson) [inline draft-ietf-tls-tls13.md]: One way to fix that is to add `{#mti-extensions}` to the end of this line.

File: draft-ietf-tls-tls13.md [hunk 2]
```diff
@@ -1459,7 +1460,7 @@ non-PSK case discussed in the previous paragraph.
 The server indicates its selected parameters in the ServerHello as
 follows:
 
-- If PSK is being used then the server will send a
+- If PSK is being used, then the server will send a
```
Comment (kaduk) [inline draft-ietf-tls-tls13.md]: I find that the RFC Editor staff have a different idea of when commas are appropriate than I do, so I try to restrain myself from making too many changes of that nature.  (Just FYI; I would probably add this if it was my document.)

File: draft-ietf-tls-tls13.md [hunk 3]
```diff
@@ -1884,7 +1885,7 @@ Cookies serve two primary purposes:
 
 - Allowing the server to offload state to the client, thus allowing it to send
   a HelloRetryRequest without storing any state. The server does this by
-  pickling that post-ClientHello hash state into the cookie (protected
+  setting the post-ClientHello hash state as the cookie (protected
```
Comment (kaduk) [inline draft-ietf-tls-tls13.md]: I don't think this is quite right, as the server can (and should) put other state into the cooke as well as just the hash state.  "pickling" is perhaps a needless term of art, though, and "including [state] in the cookie" would do just fine.

File: draft-ietf-tls-tls13.md [hunk 4]
```diff
@@ -1901,7 +1902,7 @@ which signature algorithms may be used in digital signatures. Clients which
 desire the server to authenticate itself via a certificate MUST send this extension.
 If a server
 is authenticating via a certificate and the client has not sent a
-"signature_algorithms" extension then the server MUST
+"signature_algorithms" extension then, the server MUST
 abort the handshake with a "missing_extension" alert
```
Comment (kaduk) [inline draft-ietf-tls-tls13.md]: This comma, though, I disagree with.  (Maybe it was intended to be before "then" and not after?)

File: draft-ietf-tls-tls13.md [hunk 5]
```diff
@@ -3520,7 +3521,9 @@ Application Data records may contain a zero-length TLSInnerPlaintext.content if
 the sender desires.  This permits generation of plausibly-sized cover
 traffic in contexts where the presence or absence of activity may be
 sensitive.  Implementations MUST NOT send Handshake or Alert records
-that have a zero-length TLSInnerPlaintext.content.
+that have a zero-length TLSInnerPlaintext.content, if such a message
+is recieved, the receiving implementation MUST terminate the connection
+with an "unexpected_message" alert.
```
Comment (kaduk) [inline draft-ietf-tls-tls13.md]: This adds a comma splice.  A new sentence would be fine.

File: draft-ietf-tls-tls13.md [hunk 6]
```diff
@@ -4423,7 +4426,7 @@ TLS protocol issues:
   (see {{backward-compatibility}})
 
 -  Do you handle TLS extensions in ClientHello correctly, including
-  unknown extensions.
+  unknown extensions?
```
Comment (kaduk) [inline draft-ietf-tls-tls13.md]: I have this in #810 too, but leaving it here shouldn't cause any issues.

File: draft-ietf-tls-tls13.md [hunk 7]
```diff
@@ -1036,9 +1036,10 @@ Subsequent Handshake:
 
 As the server is authenticating via a PSK, it does not send a
 Certificate or a CertificateVerify message. When a client offers resumption
-via PSK, it SHOULD also supply a "key_share" extension to the server as well
-to allow the server to decline resumption and fall back to a full handshake,
-if needed. The server responds with a "pre_shared_key" extension
+via PSK, it SHOULD also supply a "key_share" extension to the server (to provide
+forward secrecy) as well as allow the server to decline resumption and fall back
```
Comment (ekr) [inline draft-ietf-tls-tls13.md]: This seems unduly prescriptive. It's up to people whether they want to do a DH exchange here. The reason to provide a key is to allow the full handshake.

File: draft-ietf-tls-tls13.md [hunk 8]
```diff
@@ -2045,8 +2046,8 @@ CertificateAuthoritiesExtension structure.
 authorities
 : A list of the distinguished names {{X501}} of acceptable
   certificate authorities, represented in DER-encoded {{X690}} format.  These
-  distinguished names specify a desired distinguished name for a
-  root CA or for a subordinate CA; thus, this message can be used to
+  distinguished names specify a desired distinguished name for a root CA,
+  subordinate CA, or trust anchor; thus, this message can be used to
```
Comment (ekr) [inline draft-ietf-tls-tls13.md]: root CA and trust anchor are the same thing in this context.

File: draft-ietf-tls-tls13.md [hunk 9]
```diff
@@ -3520,7 +3521,9 @@ Application Data records may contain a zero-length TLSInnerPlaintext.content if
 the sender desires.  This permits generation of plausibly-sized cover
 traffic in contexts where the presence or absence of activity may be
 sensitive.  Implementations MUST NOT send Handshake or Alert records
-that have a zero-length TLSInnerPlaintext.content.
+that have a zero-length TLSInnerPlaintext.content, if such a message
```
Comment (ekr) [inline draft-ietf-tls-tls13.md]: content; if

File: draft-ietf-tls-tls13.md [hunk 10]
```diff
@@ -3520,7 +3521,9 @@ Application Data records may contain a zero-length TLSInnerPlaintext.content if
...
+is recieved, the receiving implementation MUST terminate the connection
```
Comment (ekr) [inline draft-ietf-tls-tls13.md]: i before e, except after c

File: draft-ietf-tls-tls13.md [hunk 11]
```diff
@@ -4389,8 +4392,8 @@ unsatisfactory, {{RFC4086}} provides guidance on the generation of random values
 Implementations are responsible for verifying the integrity of certificates and
 should generally support certificate revocation messages. Certificates should
 always be verified to ensure proper signing by a trusted Certificate Authority
-(CA). The selection and addition of trusted CAs should be done very carefully.
-Users should be able to view information about the certificate and root CA.
+(CA). The selection and addition of trusted anchors should be done very carefully.
```
Comment (ekr) [inline draft-ietf-tls-tls13.md]: trusted -> trust

File: draft-ietf-tls-tls13.md (hunk 3 above)
Comment (ekr) [inline draft-ietf-tls-tls13.md]: I ended up using serialize

**Comment by ekr** [comment]
Rebased and merged. @cem- please provide a PR adding yourself to contributors in the appropriate location

**Pull request closed without merging**